src/main/java/com/google/firebase/messaging/WebpushConfig.java

```diff
@@ -35,12 +35,12 @@
   private final Map<String, String> data;
 
   @Key("notification")
-  private final WebpushNotification notification;
+  private final Map<String, Object> notification;
```

Comment:
Sorry, I don't quite follow where this private fields are ever read. Can you elaborate?

I am asking this because I liked using `WebpushNotification` here better but I can see reason why the Map would be more appropriate.

Reply:
All these get serialized into JSON. We now need a Map because the notification now allows arbitrary key-value pairs (via `putCustomData()`).